src/panels/developer-tools/yaml_configuration/developer-yaml-config.ts

```diff
@@ -36,6 +36,12 @@ export class DeveloperYamlConfig extends LitElement {
 
   private _validateLog = "";
 
+  public connectedCallback() {
+    super.connectedCallback();
```

Comment:
Can we instead do this on disconnected callback?

Reply:
Any particular reason that is preferred? Either works and I am fine with either, but I only slighty thought connected might be better because it will also clear the result if someone starts validation and navigates away from the page before it completes (not sure how plausible that would be).

Reply:
Mainly for memory reasons, as it is not needed when disconnected better to clear it already

Reply:
done